ios/FluentUI/Vnext/Core/UIKit+SwiftUI_interoperability.swift

```diff
@@ -58,3 +58,23 @@ struct UIViewAdapter: UIViewRepresentable {
         stackView.addArrangedSubview(makeView())
     }
 }
+
+/// This is View wrapper that contains a UIView, View for SwiftUI/UIKit components.
+@objc public class MSFView: NSObject {
+
+    // Retrieve View for SwiftUI rendering
+    public let swiftView: AnyView
+
+    // Retrieve View for UIKit rendering
+    public let UIView: UIView
+
+    public init(_ swiftView: AnyView) {
+        self.swiftView = swiftView
+        self.UIView = UIHostingController(rootView: swiftView).view
+    }
+
+    @objc public init(_ uiView: UIView) {
+        self.UIView = uiView
```

Comment:
I think creating a UIHostingController or a UIViewAdapter anytime we create this class won't be needed in many scenarios. Which is resources we're failing to safe.